autoload/ddc/map.vim

```diff
@@ -78,6 +78,19 @@ function ddc#map#insert_item(number, cancel_key) abort
   " Call CompleteDone later.
   autocmd ddc TextChangedI * ++once doautocmd <nomodeline> CompleteDone
 
+  " Avoid side effects of ddc-cmdline
+  if mode() ==# 'c'
+    const cmdline = getcmdline()
```

Comment:
Please use `input` instead.

Reply:
I can't use `input` for this purpose because it only reads until the cursor position.

Reply:
Hm.  OK.